src/pkg/packaging/dir.proj

```diff
@@ -164,7 +164,11 @@
 
   <Target Name="GenerateCombinedInstallers" DependsOnTargets="InitPackage;GenerateBundles" />
 
-  <Target Name="GenerateNugetPackages" DependsOnTargets="InitPackage" Condition="'$(UsePrebuiltPortableBinariesForInstallers)' == 'false'">
+  <PropertyGroup>
+    <PackageManagedComponents>false</PackageManagedComponents>
+  </PropertyGroup>
+
+  <Target Name="GenerateNugetPackages" DependsOnTargets="InitPackage" Condition="'$(UsePrebuiltPortableBinariesForInstallers)' == 'false' and '$(PackageManagedComponents)' == 'true'">
```

Comment:
Yes it is fine to have them build to the bin folder we just don't want to package and ship the packages unless we need to fix something in the library.